demo-forms/mrdt-demo.xml

```diff
@@ -19,17 +19,17 @@
     </model>
   </h:head>
   <h:body class="pages">
-    <group appearance="field-list mrdt-verify" ref="/mrdt">
+    <group appearance="field-list" ref="/mrdt">
       <label>Submit a photo of an MRDT test result</label>
       <input ref="/mrdt/name">
         <label>Patient name</label>
       </input>
+      <input ref="/mrdt/mrdt_image_data" appearance="mrdt-verify">
+        <label>MRDT image</label>
+      </input>
       <input ref="/mrdt/mrdt_time_taken" appearance="mrdt-time-taken hidden">
```

Comment:
Can we append a hidden input box [here](https://github.com/medic/medic/pull/5728/files/4f0b38d6a56b479cf203746039dcae8c7393da92..e23af568dc64a458ae3773af877a113caac801e9#diff-a8375f122d90b219ecbab54fb58a8154R77) instead of relying on it to be created by the XForm author?

Reply:
I am not sure about the benefit of it since they have to update the form to add:
https://github.com/medic/medic/pull/5728/files/4f0b38d6a56b479cf203746039dcae8c7393da92..e23af568dc64a458ae3773af877a113caac801e9#diff-7d82c76043223e0e7ac50f1a5b4f9dfeR10
and
https://github.com/medic/medic/pull/5728/files/4f0b38d6a56b479cf203746039dcae8c7393da92..e23af568dc64a458ae3773af877a113caac801e9#diff-7d82c76043223e0e7ac50f1a5b4f9dfeR17

Reply:
sad face